packages/features/portfolio/src/side-drawers/messages.tsx

```diff
@@ -132,6 +132,16 @@ export const messages: Record<
       description: '',
     },
   }),
+  [PORTFOLIO_ACTIONS.COOL_DOWN]: defineMessages({
+    heading: {
+      defaultMessage: 'Cool down',
+      description: '',
+    },
+    helptext: {
+      defaultMessage: 'Cool down',
```

Comment:
Shouldn't this help text change? Or is this unused?

Reply:
Wanted to have the COOL_DOWN be able to be included in the PORTFOLIO_ACTIONS. Its unused. Just there to please the typescript gods.